src/main/java/org/ice4j/ice/Agent.java

```diff
@@ -129,6 +129,8 @@ public void run()
     private final Map<String, IceMediaStream> mediaStreams
         = new LinkedHashMap<>();
 
+    SecureRandom random = new SecureRandom();
```

Comment:
I don't think this needs SecureRandom; ThreadLocalRandom.current() should be fine.